src/main/resources/archetype-resources/pom.xml

```diff
@@ -47,7 +47,8 @@
                 <artifactId>maven-antrun-plugin</artifactId>
                 <configuration>
                     <tasks>
-                        <copy file="target/${artifactId}-${version}.jar" todir="${spigot}/plugins"/>
+                        <property name="spigot.basedir" location="${spigot}"/>

```

Comment:
Does defining `spigot.basedir` property make it an os-specific property? How would that deal with "/" in the next line?